lib/twiml/MessagingResponse.js

```diff
@@ -1,78 +1,102 @@
 'use strict';
 
-var builder = require('xmlbuilder');
+/* jshint ignore:start */
+/**
+ * This code was generated by
+ * \ / _    _  _|   _  _
+ *  | (_)\/(_)(_|\/| |(/_  v1.0.0
+ *       /       /
+ */
+/* jshint ignore:end */
+
+var builder = require('xmlbuilder');  /* jshint ignore:line */
+
 
+/* jshint ignore:start */
 /**
- * TwiML wrapper for https://www.twilio.com/docs/api/twiml/sms/your_response
- *
- * @constructor
+ * <Response> TwiML for Messages
  */
+/* jshint ignore:end */
 function MessagingResponse() {
-  this.response = builder.create('Response').dec('1.0', 'UTF-8');
+  this.response= builder.create('Response').dec('1.0', 'UTF-8');
 }
 
+/* jshint ignore:start */
 /**
- * TwiML wrapper for https://www.twilio.com/docs/api/twiml/sms/message
+ * <Message> TwiML Verb
  *
  * @param {object} attributes - ...
- * @param {string} [attributes.to] number to send message to
- * @param {string} [attributes.from] number to send from
- * @param {string} [attributes.method] (GET, POST)
- * @param {string} [attributes.action] action url
- * @param {string} [attributes.statusCallback] status callback url
- * @param {string} body body of message
+ * @param {phone_number} [attributes.to] Phone Number to send Message to
```

Comment:
since we are now telling people to use the actual types instead of just `string`, will the xml library handle serializing these types to strings in the xml?

Reply:
we don't have any types for these in node. these serve more as content hints even though you are just passing in a string.